support/Pipelines/Bbh/InitialData.py

```diff
@@ -200,6 +205,79 @@ def generate_id(
     )
 
 
+def postprocess_id(
```

Comment:
If this is where the ID root find is going to go, thoughts on putting this (and the CLI command) in a separate python file just so things aren't too cluttered? Do we have another example of having two CLIs in the same file?

Reply:
Ok yea a separate file is probably better. I'll do that when squashing.